compiler/rustc_symbol_mangling/src/v0.rs

```diff
@@ -58,39 +52,17 @@ pub(super) fn mangle<'tcx>(
     std::mem::take(&mut cx.out)
 }
 
-pub(super) fn mangle_typeid_for_fnabi<'tcx>(
```

Comment:
This needs to be kept separate from `mangle_type`. There is another open PR which completely changes the implementation of this function to use the itanium symbol mangling scheme to be compatible with clang generated CFI information.

Reply:
I just moved it, *in order to* keep it separate. I don't think it belongs in `v0`, that's all.

Reply:
Missed that.